plugins/woocommerce/changelog/fix-fix-php-docblock-for-wc-cli-runner-register-route-commands-47598

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: enhancement
+
+Fixed PHP DocBlock parameter for WC_CLI_Runner::register_route_commands
```

Comment:
```suggestion
Significance: patch
Type: fix

Updated the PHP DocBlock for WC_CLI_Runner::register_route_commands to correctly specify the first parameter's type as being a WC_CLI_REST_Command object rather than a string.
```